deepmd/pt/model/descriptor/descriptor.py

```diff
@@ -183,27 +196,13 @@ def share_params(self, base_class, shared_level, resume=False):
             # link buffers
             if hasattr(self, "mean") and not resume:
                 # in case of change params during resume
-                sumr_base, suma_base, sumn_base, sumr2_base, suma2_base = (
-                    base_class.sumr,
-                    base_class.suma,
-                    base_class.sumn,
-                    base_class.sumr2,
-                    base_class.suma2,
-                )
-                sumr, suma, sumn, sumr2, suma2 = (
-                    self.sumr,
-                    self.suma,
-                    self.sumn,
-                    self.sumr2,
-                    self.suma2,
-                )
-                stat_dict = {
-                    "sumr": sumr_base + sumr,
-                    "suma": suma_base + suma,
-                    "sumn": sumn_base + sumn,
-                    "sumr2": sumr2_base + sumr2,
-                    "suma2": suma2_base + suma2,
-                }
+                base_env = EnvMatStatSeA(base_class)
+                for kk in base_class.get_stats():
+                    base_env.stats[kk] += self.get_stats()[kk]
```

Comment:
`base_env.stats` is still empty at the begining, rather than start from `base_class.get_stats()`?

Reply:
Fixed in https://github.com/deepmodeling/deepmd-kit/pull/3285/commits/7c9a66e544a371cc33dae7a03e4e7a7ebf66d950